src/main/webapp/script.js

```diff
@@ -264,10 +287,12 @@ function getTooltipContent(node) {
 function navigateGraph(amount) {
   currGraphNum += amount;
   if(currGraphNum < 0) {
-    currGraphNum = 0;
+    setCurrGraphNum(0);
```

Comment:
I don't think you need to call this function - you can just set currGraphNum directly. 
Alternatively, you can make the setCurrGraphNum function clamp the values to 0 and numMutations, and just call `setCurrGraphNum(currGraphNum + amount)`